src/test/scala/is/hail/methods/LDPruneSuite.scala

```diff
@@ -0,0 +1,285 @@
+package is.hail.methods
+
+import breeze.linalg.{Vector => BVector}
+import is.hail.SparkSuite
+import is.hail.check.Prop._
+import is.hail.check.{Gen, Properties}
+import is.hail.variant._
+import is.hail.utils._
+import org.testng.annotations.Test
+
+class LDPruneSuite extends SparkSuite {
+  val bytesPerCore = 256L * 1024L * 1024L
+
+  def convertGtToGs(gts: Array[Int]): Iterable[Genotype] = gts.map(Genotype(_)).toIterable
+
+  def toNormalizedGtArray(gs: Array[Int], nSamples: Int): Option[Array[Double]] = {
```

Comment:
I tried to use `RegressionUtils.toNormalizedGtArray` but it gives a different answer. I think the normalization must be different.